### PR TITLE
GPU Documentation Updates

### PR DESCRIPTION
**Pull request by vadam5**
Updated Replicate Alpaca doc into include GPU instructions and made some minor changes to GPU installation docs.

### CODE REVIEW

File: docs/Getting-Started-GPU.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
-# Getting Started on CUDA
```
Comment (vadam5) [inline docs/Getting-Started-GPU.md:1]: Changed CUDA to GPU for symmetry with TPU in the docs

File: docs/Getting-Started-GPU.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
+**Note** that right now GPU support is very preliminary and Levanter does not work on a multi-machine setup. (We think this is just a dependency/environment issue.)
 
 We only test on Ampere (e.g. A100s or 30xx series)  GPUs. If it works with Jax it should work, though.
 
 ## Installation
 ### Setting up Conda Environment
 ```bash
 # 3.11 is too new for tensorstore
-# this doesn't actually install pytorch, but it bundles cuda dependencies nicely
-conda create --name levanter -c pytorch -c nvidia pytorch-cuda=11.7 python~=3.10
```
Comment (vadam5) [inline docs/Getting-Started-GPU.md:12]: including `-c pytorch and -c nvidia pytorch-cuda=11.7` was causing issues when I was installing jax with cuda and then when I was installing levanter, but without them everything seems to be running correctly.

File: docs/Getting-Started-GPU.md [hunk 3]
```diff
@@ -27,18 +26,28 @@ pip install --upgrade "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-re
...
+
 ## Running a job on Slurm
 
 ### Single Node
 
-Simple example, customize to your needs:
+A simple srun example within your levanter conda environment and cloned levanter directory. Customize to your needs:
 
 ```bash
-srun --cpus-per-task=128 --gres=gpu:8 --job-name=levanter-multi-1 --mem=1000G --nodelist=sphinx7 --open-mode=append --partition=sphinx --time=14-0 ~/src/levanter/scripts/run-slurm.sh python src/levanter/main/train_lm.py --config_path config/gpt2_small.yaml
```
Comment (vadam5) [inline docs/Getting-Started-GPU.md:41]: This command was also giving me some issues and I thought an example from the examples dir might be fitting here. Especially now that we'll have a whole doc on customizing this example to the user's needs.

File: docs/Installation.md [hunk 4]
```diff
@@ -1,33 +1,6 @@
 # Installation
 
-## Quickstart
```
Comment (vadam5) [inline docs/Installation.md:3]: Basically switched up the order of information here to follow the order someone getting setup for the first time might follow. Still not sure if this is the best flow.

File: docs/Getting-Started-GPU.md (hunk 3 above)
Comment (vadam5) [reply docs/Getting-Started-GPU.md:41]: Okay, I'll switch it back

File: docs/Installation.md [hunk 5]
```diff
@@ -1,33 +1,6 @@
...
-Now clone this repository and install it with pip:
-
-```bash
-git clone https://github.com/stanford-crfm/levanter.git
-cd levanter
-pip install -e .
-wandb login  # optional, we use wandb for logging
-```
-
+If you are using GPUs, please follow the installation steps [here instead](Getting-Started-GPU.md).
```
[the remark this answers is not in the document]
Comment (vadam5) [reply docs/Installation.md:3]: I thought the setup instructions below needed to be followed before TPU setup, but I'll just move it to be by GPU so people don't miss it.

File: docs/Getting-Started-GPU.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+## Installation
+### Setting up Conda Environment
+```bash
+# 3.11 is too new for tensorstore
+conda create --name levanter python=3.10
+conda activate levanter
+```
+### Install Jax with CUDA
+
+This is a bit in flux in Jax, but please take a look at the [Jax Installation Guide](https://github.com/google/jax#pip-installation-gpu-cuda-installed-via-pip-easier). Here are two options that work circa June 2023:
```
Comment (dlwh) [inline docs/Getting-Started-GPU.md:16]:
```suggestion
This is a bit in flux in JAX, but please take a look at the [JAX Installation Guide](https://github.com/google/jax#pip-installation-gpu-cuda-installed-via-pip-easier). Here are two options that work circa June 2023:
```

File: docs/Getting-Started-GPU.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+### Install Levanter
+
+```bash
+git clone https://github.com/stanford-crfm/levanter.git
+cd levanter
+pip install -e .
+```
+
+### wandb login
```
Comment (dlwh) [inline docs/Getting-Started-GPU.md:34]:
```suggestion
### WandB login
```

File: docs/Getting-Started-GPU.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+To use wandb to track your learning curves and other experiment metrics, you con log in to your wandb account on the command line as follows:
```
Comment (dlwh) [inline docs/Getting-Started-GPU.md:36]:
```suggestion
To use WandB to track your learning curves and other experiment metrics, you con log in to your WandB account on the command line as follows:
```

File: docs/Getting-Started-GPU.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+```bash
+wandb login ${YOUR TOKEN HERE}
+```
+You can find more information on getting setup with weights and biases here: https://wandb.ai/site
```
Comment (dlwh) [inline docs/Getting-Started-GPU.md:40]:
```suggestion
You can find more information on getting setup with Weights and Biases here: https://wandb.ai/site
```

File: docs/Installation.md [hunk 10]
```diff
@@ -51,8 +26,24 @@ Using [Virtualenv](https://docs.python.org/3/library/venv.html#creating-virtual-
...
 # Activate the virtual environment.
 source levanter-venv/bin/activate
 ```
+
+## Setting up a development environment
+
+For development, please follow these steps:
+
+First, install the appropriate version of Jax for your system.
+See [Jax's installation instructions](https://github.com/google/jax/blob/main/README.md#installation)
```
Comment (dlwh) [inline docs/Installation.md:40]:
```suggestion
See [JAX's installation instructions](https://github.com/google/jax/blob/main/README.md#installation)
```

**Review by dlwh — APPROVED** [review]
couple of nits you can bulk commit if you want. lgtm!

**Pull request merged**